### PR TITLE
Updated the bbb-webhooks documentation. Ensuring the configuration file gets ignored.

### PR DESCRIPTION
**Pull request by AlexJezior**
The current bbb-webhooks `readme.md` documentation doesn't accurately tell how to get the bbb-webhooks configured for custom development, and release procedures. Updates are my venture into configuring webhooks for my own personal use, while providing documentation to help others in the future from having to start a new topic in regards, on the bigbluebutton-dev group forum.

### CODE REVIEW

File: bbb-webhooks/README.md [hunk 1]
```diff
@@ -1,97 +1,72 @@
-bbb-webhooks
-============
+# bbb-webhooks
 
 This is a node.js application that listens for all events on BigBlueButton and sends POST requests with details about these events to hooks registered via an API. A hook is any external URL that can receive HTTP POST requests.
 
-You can read the full documentation at: https://docs.bigbluebutton.org/dev/webhooks.html
+You can read the full documentation at: http://docs.bigbluebutton.org/labs/webhooks.html
```
Comment (fcecagno) [inline bbb-webhooks/README.md:5]: The original URL was correct.

File: bbb-webhooks/README.md [hunk 2]
```diff
@@ -1,97 +1,72 @@
...
+            {
+                url: 'https://staging.example.com/webhook-post-route',
+                getRaw: false
+            },
+            {
+                url: 'https://app.example.com/webhook-post-route',
+                getRaw: false
+            }
+        ]
+      ```
```
Comment (fcecagno) [inline bbb-webhooks/README.md:45]: Two suggestions:
- permanentURLs is above the hooks attribute, so I suggest to modify the sentence to something like "Locate hooks.permanentURLs on config/default.yml and modify it as follows:"
- use the yaml syntax as follows:
```
permanentURLs:
  - url: 'https://staging.example.com/webhook-post-route'
    getRaw: false
  - url: 'https://app.example.com/webhook-post-route'
    getRaw: false
```

File: bbb-webhooks/README.md [hunk 3]
```diff
@@ -1,97 +1,72 @@
...
-~~~
-wget http://nodejs.org/dist/vX.X.X/node-vX.X.X.tar.gz
-tar -xvf node-vX.X.X.tar.gz
-cd node-vX.X.X/
-./configure
-make
-sudo make install
-~~~
+If you are editing these permanent urls after they have already been committed to the application once, you may need to flush the redis database in order for adjustments to these permanent hooks to get picked up by your application. Use the following command to do so:
+ - `redis-cli flushall`
```
Comment (fcecagno) [inline bbb-webhooks/README.md:50]: Please add a warning mentioning that all events will be wiped from redis, which means that no ongoing recording will be processed.

File: bbb-webhooks/.gitignore [hunk 4]
```diff
@@ -5,3 +5,4 @@ node_modules/
 config_local.coffee
 config_local.js
```
Comment (fcecagno) [inline bbb-webhooks/.gitignore:6]: Would you please remove these two, since they are no longer used?

**Review by fcecagno — CHANGES_REQUESTED** [review]
Good job @AlexJezior, would you please take a look at my comments? Thanks!

File: bbb-webhooks/README.md (hunk 1 above)
Comment (AlexJezior) [reply bbb-webhooks/README.md]: Interesting! I don't even recall making this change; should be resolved now.

File: bbb-webhooks/README.md (hunk 3 above)
Comment (AlexJezior) [reply bbb-webhooks/README.md:46]: Great idea!

File: bbb-webhooks/.gitignore (hunk 4 above)
Comment (AlexJezior) [reply bbb-webhooks/.gitignore]: Almost did this originally, but wasn't sure if there would be potential lingering configurations floating around out there that we would still like to keep out of the repository.

**Review by fcecagno — APPROVED** [review]
Thank you for your contribution.

**Comment by antobinary** [comment]
@AlexJezior Thank you for your contribution! Could you please confirm you have filled out a CLA? https://docs.bigbluebutton.org/support/faq.html#why-do-i-need-to-sign-a-contributor-license-agreement-to-contribute-source-code ?

**Comment by AlexJezior** [comment]
@fcecagno @antobinary My pleasure to help!

I did fill out and email the CLA yesterday to the ffdixon@bigbluebutton.org email address.

**Pull request merged**